src/transformers/modeling_lxmert.py

```diff
@@ -400,15 +502,15 @@ def forward(self, lang_feats, lang_attention_mask, visn_feats, visn_attention_ma
         visn_att_output = (visn_feats,)
 
         lang_att_output, visn_att_output = self.cross_att(
-            lang_att_output[0], lang_attention_mask, visn_att_output[0], visn_attention_mask,
+            lang_att_output[0], lang_attention_mask, visn_att_output[0], visn_attention_mask, output_x_attentions=output_attentions
         )
         attention_probs = lang_att_output[1:]
         lang_att_output, visn_att_output = self.self_att(
             lang_att_output[0], lang_attention_mask, visn_att_output[0], visn_attention_mask
         )
 
         lang_output, visn_output = self.output_fc(lang_att_output, visn_att_output)
-        return (lang_output, visn_output) + (attention_probs,) if output_attentions else (lang_output, visn_output)
+        return (lang_output, visn_output) + (attention_probs[0],) if output_attentions else (lang_output, visn_output)
```

Comment:
Here `attention_probs = lang_att_output[1:]`, with `len(lang_att_output)` in [0, 1]. 

If 1, then `lang_att_output[1:]` returns a tuple containing a single variable, and we then concatenate that tuple to the return tuple, resulting in a tuple containing a tuple of a tensor, instead of a tuple containing only tensors.